QuEST/src/QuEST_validation.c

```diff
@@ -679,6 +681,29 @@ void validateDiagonalOp(Qureg qureg, DiagonalOp op, const char* caller) {
     QuESTAssert(qureg.numQubitsRepresented == op.numQubits, E_MISMATCHING_QUREG_DIAGONAL_OP_SIZE, caller);
 }
 
+void validateIsIsingSpinHamiltonian(PauliHamil hamil, const char* caller){
```

Comment:
name should be left agnostic of physical system the form happens to be able to model (should not reference Ising)